src/components/achievements/Stat.astro

```diff
@@ -25,17 +26,22 @@ const repoCounts = Object.entries(stat).sort(([, a], [, b]) => b - a);
     <H class="heading-5 text-white">{type[0].toUpperCase() + type.slice(1)}</H>
     <p class="text-[2rem] text-white font-obviously-wide font-semibold leading-none">{count}</p>
   </div>
-  <ul class="text-[13px] text-white leading-tight">
+  <ul class="text-[13px] leading-tight">
     {
       repoCounts.map(([repo, repoCount]) => {
         const percentage = (repoCount / maxCount) * 100;
         return (
           <li class="grid grid-cols-2 items-center">
-            <span
-              class="text-right py-0.5 pr-2 border-r border-neutral-500 break-words"
+            <a
+              href={`https://github.com/withastro/${repo}/${
+								type === 'issues' ? 'issues' : 'pulls'
+							}?q=is:${type === 'issues' ? 'issue' : 'pr'}+${
+								type === 'reviews' ? 'reviewed-by' : 'author'
+							}:${username}`}
+              class="text-right py-0.5 pr-2 border-r border-neutral-500 break-words hover:text-white"
```

Comment:
The Astro Tailwind kit includes a `link` class that should work well here. Let's us keep the text white but includes standard hover styles.
```suggestion
              class="text-right py-0.5 pr-2 border-r border-neutral-500 break-words link"
```